panel/template/fast/list/fast_list_template.html

```diff
@@ -54,6 +54,11 @@
       box-shadow: 2px 2px 10px silver;
   }
   {% endif %}
+  {% if style.maximize_panels %}
+  .bk-root {
+    height: calc( 100vh - 100px ) !important;
```

Comment:
This scares me, you certainly don't want all roots to be full screen height right?

Reply:
My intention is to maximize all roots in the main area. I can see I do more and will adjust.

The alternative is to add functionality to maximize the first root only.

If I look at my apps awesome-panel.org and at work most of these are are should just have one maximized main root. Its such a bad user experience currently where a main root is around 500px by default or alternatively some fixed size specified by the user. 

Users come complaining it does not fit their big trader screen, their small laptop screen or a custom window size they need to fit it in their complex set of windows. Developers come complaining that laying out apps is too difficult in Panel.

Reply:
I don't see and option that is `False` by default as very scary. But it would make it so much easier to create apps that layout nicely.

Reply:
I've changed the selector to `.main .bk-root`.